scoresheets/CleanTable.tex

```diff
@@ -2,14 +2,21 @@
 
 \begin{scorelist}
 	\scoreheading{Main Goal}
-	\scoreitem[5]{60}{Picking tableware and cutlery for transportation to the dishwasher}
-	\scoreitem[5]{180}{Placing the tableware and cutlery inside the dishwasher}
+	\scoreitem{15}{Navigate to the table to pick up items}
```

Comment:
Should this be scored? It is very simple for a stage 2 challenge.

Reply:
Yes, I think we decided it is fine to award small amounts of points for things like this. It motivates teams and also differentiates teams that enter and attempt the test from teams that don't attempt the test at all.